assets/js/src/newsletters/types/re_engagement/scheduling.tsx

```diff
@@ -25,6 +25,9 @@ export function Scheduling({
 }: Props): JSX.Element {
   const daysInPeriod = afterTimeType === 'weeks' ? 7 : 30;
   const daysSelected = Number(afterTimeNumber) * daysInPeriod;
+  let inactivePeriod = inactiveSubscribersPeriod;
+  // so that we can calculate the dates using
```

Comment:
I don't understand this comment. It feels like part of it is missing. I see from the commit message that it has something to do with `multiplication of 30` but I don't get why we need it. Could you please update the comment. Also it seems that if a user selects 361 days they won't see the warning. But I guess that is a tradeoff we could accept.